compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/SkiaRootForTest.skiko.kt

```diff
@@ -32,6 +33,11 @@ interface SkiaRootForTest : RootForTest {
      */
     val scene: ComposeScene get() = throw UnsupportedOperationException("SkiaRootForTest.scene is not implemented")
 
+    /**
+     * Whether or not this root has been placed in the hierarchy.
```

Comment:
Better to describe it more, when it is not placed, and when it becomes placed. Is it not placed until we layout the content? I.e. from `ComposeScene` perspective does it become placed when we call `render`? If yes, I know the side effect - when we will call `contentSize` it will also become placed, and I am not sure that it is good. Maybe we just don't need this property, and just don't check it in `checkIsDisplayed`, but I am not sure.

Reply:
You're right, it becomes placed as soon as SkiaBasedOwner is initialized (in `root.attach(this)`).
So I'll remove that check for now.